core/src/main/scala/org/apache/spark/sql/SnappySession.scala

```diff
@@ -1852,7 +1876,7 @@ object SnappySession extends Logging {
 
     if (key ne null) {
       val noCaching = session.getContextObject[Boolean](
-        CachedPlanHelperExec.NOCACHING_KEY).isDefined
+        SnappyParserConsts.NOCACHING_KEY).isDefined
```

Comment:
Same as above. As in moving the constants does not look too necessary.

Reply:
Not necessary, but good for consolidation. Some of these are used by all of parser/session/plan (WRAPPED_CONSTANTS_KEY) while others by session/plan. The WRAPPED_CONSTANTS_KEY was the one that was moved since its used by parser, but then moved others too for consistency.